mods/example/maps/blank-map/map.yaml

```diff
@@ -17,10 +17,5 @@ Visibility: Shellmap, Lobby
 Categories: Maps
 
 Players:
-	PlayerReference@Neutral:
```

Comment:
I would expect this to crash on launch, since we don't have an owner for the World actor.  Can we fix this in a different way (and then file an engine bug so that it *does* crash)?